integration/kubernetes/operator/alluxio/go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/onsi/gomega v1.4.2
 	github.com/pkg/errors v0.8.1
 	go.uber.org/zap v1.9.1
+	golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2 // indirect
```

Comment:
is this needed?

Reply:
This line keeps coming up automatically in my repo...